sdk/textanalytics/Azure.AI.TextAnalytics/src/CategorizedEntity.cs

```diff
@@ -13,18 +13,12 @@ namespace Azure.AI.TextAnalytics
     /// </summary>
     public readonly struct CategorizedEntity
     {
-        internal CategorizedEntity(string text, string category, string subCategory, double score)
-        {
-            Text = text;
-            Category = category;
-            SubCategory = subCategory;
-            ConfidenceScore = score;
-        }
-
         internal CategorizedEntity(Entity entity)
         {
-            Text = entity.Text;
+            // We shipped TA 5.0.0 Text == string.Empty if the service returned a null value for Text.
+            // Because we don't want to introduce a breaking change, we are transforming that null to string.Empty
             Category = entity.Category ?? string.Empty;
+            Text = entity.Text;
```

Comment:
According to the comment, we should be doing:
```cs
Text = entity.Text ?? string.Empty;
```
as well, shouldn't we?

Reply:
hah! thank you!
This was my error. It should have said Category.